requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
-pytest==7.4.2
+pytest==8.3.3
 mock==3.0.5
 pexpect==3.3
 twine==1.13.0
-tox>=1.9.2
\ No newline at end of file
+tox>=1.9.2
+pytest-vcr>=1.0.2
```

Comment:
[VCR](https://vcrpy.readthedocs.io/en/latest/) dep lets us add serverless tests without supplying a serverless endpoint, it's behind the new `casette` file.